code/script.js

```diff
@@ -30,18 +38,137 @@ const showMessage = (message, sender) => {
   chat.scrollTop = chat.scrollHeight
 }
 
-// Starts here
+// Starts here, all the bot questions
 const greeting = () => {
+  currentQuestion = 1
   showMessage(`Hello there, What's your name?`, 'bot')
-  // Just to check it out, change 'bot' to 'user' here 👆
+  console.log('bot')
+ }
+
+const oreoQuestion = () => {
+  currentQuestion++
+  setTimeout(() => 
+  showMessage('How do you eat an Oreo?','bot'), 1200);
+  
+  // Generate a set of buttons with bite choices
+  inputWrapper.innerHTML = `
+  <button id="oneBiteBtn">In one bite</button>
+  <button id="pickApartBtn">Pick apart</button>
+  <button id="dipMilkBtn">Dip in milk</button>
+  `
+  // Send to next question depending on which button was clicked
+  document.getElementById('oneBiteBtn').addEventListener('click', () => {
+    showMessage('I eat it in one bite.', 'user')
+    setTimeout(() => showMessage('Excellent choice!', 'bot'), 1000)
+    handleInput()
+  })
+  document.getElementById('pickApartBtn').addEventListener('click', () => {
+    showMessage('I like to pick it apart.', 'user')
+    setTimeout(() => showMessage('Great choice!', 'bot'), 1000)
+    handleInput()
+  })
+  document.getElementById('dipMilkBtn').addEventListener('click', () => {
+    showMessage('I like to dip it in milk.', 'user')
+    setTimeout(() => showMessage('Wet and fantastic choice!', 'bot'), 1000)
+    handleInput()
+  })
+
+  console.log('hej hej')
+
+}
+
+//Final question of our bot
+const lastQuestion = () => {
+
+  currentQuestion++
+  setTimeout(() => showMessage('Do you share your dessert with others?', 'bot'), 3000) 
+  console.log('hallu')
+  inputWrapper.innerHTML = `
+    <button id="yes1">Yes of course!</button>
+    <button id="little2">Just a small bite</button>
+    <button id="no3">Hell no!</button>
```

Comment:
It would be nice if the buttons for the answer activates when this question is shown, and not before.